src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -1,19 +1,47 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable function-paren-newline */
+/* eslint-disable object-curly-newline */
+/* eslint-disable implicit-arrow-linebreak */
+/* eslint-disable no-confusing-arrow */
+/* eslint-disable arrow-parens */
+/* eslint-disable no-undef */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable react/jsx-no-duplicate-props */
+/* eslint-disable no-multi-str */
+/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/alt-text */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable no-empty */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
+/* eslint-disable no-use-before-define */
+/* eslint-disable radix */
+/* eslint-disable no-shadow */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable jsx-a11y/scope */
+/* eslint-disable react/no-array-index-key */
```

Comment:
Why all these commented lines were added? They were added to other files too but I don't see the reason.

Reply:
It's added by the quick-fix to hide the error from eslint. Most of the errors are not errors, just good practice suggestions by eslint that I can't fix in code that was written by someone else. things like keeping one line of code under 100 characters and others, if I change it I might break the application since it would be necessary even rename variables or functions.